utils/openchangeclient.c

```diff
@@ -895,6 +900,13 @@ static enum MAPISTATUS openchangeclient_sendmail(TALLOC_CTX *mem_ctx,
 		}
 	}
 
+	/* Send the submit time */
+	unix_to_nt_time(&nt, time(NULL));
+	now.dwLowDateTime = (nt << 32) >> 32;
+	now.dwHighDateTime = (nt >> 32);
+	set_SPropValue_proptag(&props[prop_count - 1], PR_CLIENT_SUBMIT_TIME, (void *)&now);
```

Comment:
Any specific reason why you chose not to use `set_SPropValue_proptag_date_timeval` from libmapi/property.c?